### PR TITLE
[37251] Allow smaller height modals and fix title for password modal

### PR DESCRIPTION
**Pull request by oliverguenther**
https://community.openproject.org/wp/37251

### CODE REVIEW

**Comment by oliverguenther** [comment]
@b12f I added another modified to avoid the `min-height: 500px` as it's too large for the one-field password confirmation. I'm not set on this approach, please let me know what you think.

To reproduce / reach this modal:

Go to my account, change your user's e-mail address, hit save.

File: frontend/src/app/modules/modal/modal.sass [hunk 1]
```diff
@@ -27,6 +27,9 @@
     width: 70rem
     min-height: 40vh
 
+  &_fill_height
```
Comment (b12f) [inline frontend/src/app/modules/modal/modal.sass]: maybe `autogrow` or `autoheight`? I'm okay with the current naming, but in that case it should be `fill-height`, so you don have a modifier on top of a modifier

Comment (oliverguenther) [reply frontend/src/app/modules/modal/modal.sass]: I changed it to `autoheight`

**Review by b12f — APPROVED** [review]
lgtm

**Pull request merged**